src/examples/libpmemobj/string_store_tx_type/writer.c

```diff
@@ -57,7 +57,12 @@ main(int argc, char *argv[])
 	}
 
 	char buf[MAX_BUF_LEN];
-	if (scanf("%9s", buf) == EOF) {
+	int num = scanf("%9s", buf);
+	/*
+	 * added zero-terminator to prevent C6054
+	 */
+	buf[MAX_BUF_LEN - 1] = '\0';
```

Comment:
This is not necessary, quoting man 3 scanf:

> String  input  conversions store a terminating null byte ('\0') to mark the end of the input

And please don't quote windows specific numbers in the common source code.
If you insist on zeroing the buffer, use  `... = {0};` syntax on the entire array instead.